### DISCUSSION TITLE
Attiny3216 analogReference

### DISCUSSION THREAD

**Discussion by Konstantinnios** [SpenceKonde/megaTinyCore#1147] [Q&A] [answered]
Hello,
I'm trying to run analogReference on Attiny 3216 but it doesn't work. I don't know where I am wrong.
megaTinyCore 2.6.10 version
`
void setup() {
analogReference(INTERNAL2V5);
}
void loop() {
}`

I've tried all possible options for setting the reference voltage, but to no avail. I measure on pin 3 (PA5) against minus - there is no voltage whatsoever. My multimeter works - I measure the input voltage correctly. I don't know where I'm wrong, what I'm missing.

**Comment by hmeijdam** [comment, marked as answer]
I get the feeling you think the internal reference voltages are presented on the VREFA pin (PA5)

![image](https://github.com/user-attachments/assets/9464b7e5-2f92-4d8c-b5bd-72a97f735de9)
 
But that's only a analog input to connect an external reference voltage to.

**Comment by Konstantinnios** [comment]
Oh, that's right. I remembered that I have projects where I use it, and yes... that's right. It's good to start something with a fresh mind.
